### PR TITLE
baldur buff

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
baldur buff

## Why It's Good For The Game
kuro keeps screaming at me about it

## Changelog
:cl:
balance: baldur mk1 slowdown reduced and light range slightly higher, mk2 has no slowdown
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -350. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Kuro-ai** [comment]
so now baldur isn't a literal noob trap,nice

**Comment by RuneQuester** [comment]
-350 rating

**Comment by David-Stormwell** [comment]
> 
> 
> so now baldur isn't a literal noob trap,nice

uhuh sure~

**Comment by Kuro-ai** [comment]
> > so now baldur isn't a literal noob trap,nice
> 
> Never has been 🔫

always was,ever since it got nerfed

File: code/modules/modular_armor/attachables.dm [hunk 1]
```diff
@@ -51,7 +51,8 @@
 /obj/item/armor_module/attachable/better_shoulder_lamp/mark1
 	name = "\improper Mark 1 Baldur Light Amplification System"
 	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
-	power_boost = 3 /// The boost to armor shoulder light
+	power_boost = 4 /// The boost to armor shoulder light
```
Comment (TiviPlus) [inline code/modules/modular_armor/attachables.dm:54]: IIRC if you go any higher than 3 it starts causing popin so revert this please

Comment (SplinterGP) [reply code/modules/modular_armor/attachables.dm:54]: i mean the MK2 has 5

Comment (TiviPlus) [reply code/modules/modular_armor/attachables.dm:54]: ok that sfine then

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-358). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**